tools/make/common.mk

```diff
@@ -39,6 +39,7 @@ REV=$(shell git rev-parse --short HEAD)
 
 # Supported Platforms for building multiarch binaries.
 PLATFORMS ?= darwin_amd64 darwin_arm64 linux_amd64 linux_arm64 windows_amd64 windows_arm64
+IMAGE_PLATFORMS ?= ${PLATFORMS}
```

Comment:
curious why this is needed ?
the default seems to be `IMAGE_PLATFORMS ?= linux_amd64 linux_arm64` in L14 in image.mk

Reply:
As description said, although we only build linux image as default, go binary will build for all arches :(

Reply:
im still unable to understand why `IMAGE_PLATFORMS` needs to be set here ? so we can build all image types by default ?

Reply:
If not set here, build image with default value will build:
- binaries for all arch
- images only for Linux

I think should only build binaries only for Linux as well, WDYT?

Reply:
sure, how does setting L42 to all values in L41 solve this though

Reply:
In this way, the default behavior will keep sync.